homeassistant/components/waze_travel_time/const.py

```diff
@@ -21,7 +19,9 @@
 DEFAULT_AVOID_SUBSCRIPTION_ROADS = False
 DEFAULT_AVOID_FERRIES = False
 
-UNITS = [CONF_UNIT_SYSTEM_METRIC, CONF_UNIT_SYSTEM_IMPERIAL]
+UNITS_IMPERIAL = "imperial"
+UNITS_METRIC = "metric"
+UNITS = [UNITS_METRIC, UNITS_IMPERIAL]
```

Comment:
Since this is linked to config-flow and stored/used internally I think it is better to use internal constants.

Reply:
Cancelled this change